hack/make-rules/test.sh

```diff
@@ -82,6 +82,10 @@ fi
 # Set to 'y' to keep the verbose stdout from tests when KUBE_JUNIT_REPORT_DIR is
 # set.
 KUBE_KEEP_VERBOSE_TEST_OUTPUT=${KUBE_KEEP_VERBOSE_TEST_OUTPUT:-n}
+# Set to 'y' to skip normalizing the test targets. Normalization
+# is redundant for local paths starting with a . (./pkg/...) or Go package
```

Comment:
could we just automatically skip this when we notice local paths that don't need normalizing, and forget adding test tags etc?

Reply:
IE ~~`if all $WHAT start with "./" or "k8s.io/" then; SKIP_NORMALIZATION=${SKIP_NORMALIZATION:-true}` (but in valid bash)

Reply:
What I really want is just a fast way to normalize and to exclude packages without complicating thins for developers.

In bazel we could do something like this: `bazel test //... -//cmd/kubeadm/integration/...`

Maybe there's a way to do this faster than the current `find` implementation